Assets/Scripts/Editor.cs

```diff
@@ -395,11 +425,11 @@ void Update ()
 			{
 				case InteractionType.Image:
 				{
-					var editor = interactionEditor.GetComponent<ImagePanelEditor>(); 
+					var editor = interactionEditor.GetComponent<ImagePanelEditor>();
 					if (editor.answered)
 					{
 						var folder = Path.Combine(Application.persistentDataPath, meta.guid.ToString());
-						var filename = "extra" + meta.extraCounter++;
+						var filename = Path.Combine("extra", GenerateExtraGuid());
```

Comment:
Instead of Path.Combine, you could put the path in `SaveFile.cs` as a static string (next to `metaFilename`, `videoFilename`, ...)